opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -261,6 +261,9 @@ def end(self):
         if self.end_time is None:
             self.end_time = util.time_ns()
 
+    def update_name(self: 'Span', name: str) -> None:
```

Comment:
```suggestion
    def update_name(self: Span, name: str) -> None:
```

Not sure if that also works in the API package, but here it should definitely work without quotes.

Reply:
👍 You are right AFAIK, `self` is implicitly annotated with the containing class.